### DISCUSSION TITLE
Orderer Node version and Peer Node version

### DISCUSSION THREAD

**Discussion by salimbene** [hyperledger/fabric#4077] [General]
I'm presented with a fabric network that runs orderer nodes at 2.4.5, and peer nodes at 2.4.0. Is this correct? I cannot find information anywhere regarding whether the ordered and peer nodes within the same org should run at the same level. I guess it can run, but isn't it recommended?

**Comment by joarder97** [comment]
Its technically possible, but it is not recommended to have orderer nodes and peer nodes running at different versions in a Fabric network due to potential compatibility issues and inconsistencies. It is advisable to align the versions of all components for optimal operation.
